build.gradle

```diff
@@ -555,17 +553,17 @@ project(':iceberg-delta-lake') {
     // The newest version of delta-core uses Spark 3.3.*. Since its only for test, we do
     // not need to include older version of delta-core
     if (sparkVersions.contains("3.3")) {
-      integrationImplementation "io.delta:delta-core_${scalaVersion}"
+      integrationImplementation "io.delta:delta-core_${scalaVersion}:${libs.versions.deltaCore.get()}"
       integrationImplementation project(path: ":iceberg-spark:iceberg-spark-3.3_${scalaVersion}")
-      integrationImplementation("org.apache.hadoop:hadoop-minicluster") {
+      integrationImplementation(libs.hadoop2.minicluster) {
         exclude group: 'org.apache.avro', module: 'avro'
         // to make sure netty libs only come from project(':iceberg-arrow')
         exclude group: 'io.netty', module: 'netty-buffer'
         exclude group: 'io.netty', module: 'netty-common'
       }
       integrationImplementation project(path: ':iceberg-hive-metastore')
       integrationImplementation project(path: ':iceberg-hive-metastore', configuration: 'testArtifacts')
-      integrationImplementation("org.apache.spark:spark-hive_${scalaVersion}:3.3.2") {
+      integrationImplementation(libs.spark.hive33) {
```

Comment:
why don't we need a scalaversion here but require one for `io.delta:delta-standalone_${scalaVersion}`?

Reply:
Can't find this peace anymore, it seems it is outdated, hive33 mention now do use scalaVersion parameter